.env.development

```
@@ -1 +1 @@
-SITE_URL="https://elastic-banach-3b73d5.netlify.com"
+HOST="https://elastic-banach-3b73d5.netlify.com"
```

This seems this will be the default for local development, since we don't have the travis Variables there, right?
I think its then better to have:
```suggestion
HOST="http://localhost:8000"
```

---

yeah, changing it